src/pipeline.ml

```diff
@@ -75,11 +97,14 @@ let or_die = function
 
 (* Pipeline to build the opam base image and the compiler images for a particular architecture. *)
 module Arch = struct
+  (* 2020-04-29: On Windows, squashing images is still experimental (broken). *)
+  let squash = Sys.unix
```

Comment:
I think this isn't right, since the base-images _host_ will be Linux and so Sys.unix will be always set. It needs to tell the system it's dispatching to is a Linux or Windows distro and decide like that.

Reply:
Well spotted, thank you! Changed to use the OS family of the current distro.